client/components/command-palette/use-command-palette.tsx

```diff
@@ -73,9 +84,17 @@ interface SiteToActionParameters {
 	};
 }
 
+function isCustomDomain( siteSlug: string | null | undefined ): boolean {
+	if ( ! siteSlug ) {
+		return false;
+	}
+	return ! siteSlug.endsWith( '.wordpress.com' ) && ! siteSlug.endsWith( '.wpcomstaging.com' );
+}
```

Comment:
This is copied from https://github.com/Automattic/wp-calypso/blob/0cef65777daf38385989d91c46154317975a6694/client/sites-dashboard/utils.ts#L42-L47